HtmlEditor: Update Tables demo description (#1477)
DevExpress/devextreme-demos: JSDemos/Demos/HtmlEditor/Tables/description.md

> DevExtreme HTML/Markdown Editor allows its users to work with tables. This demo illustrates table resize support and table management with the toolbar, context menu, and from code.

Reviewer: > DevExtreme HTML/Markdown Editor allows **its** users to work with tables.

Was it absolutely necessary to specify that users "belong" to the Editor? The pronoun can safely be omitted without losing any sense you intended to convey.

> and **from** code

Do you always write like this? Normally, if we need a synonym to something done programmatically, we say "in code", not "from code".  However, if you always write like the way you did and it caused no issues in the past, you may simply ignore my comment.

## Manage Tables with the Toolbar

To allow users to manage tables with the [toolbar](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/), add the following [items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/) to it:

> - *"insertTable"* / *"deleteTable"* - Manage a table

Reviewer: I don't know where this tradition of breaking the sentence structure with dashes and hyphens came from but I don't like it at all. Do you always write like that? Haven't anyone ever told you that this is far from being the best approach possible?

Look. It's extremely easy to create a fully functional sentence from this strange, broken line:

```suggestion
- *"insertTable"* and *"deleteTable"* manage a table.
```
You may also want to review the following topic: [Lists](https://tw.devexpress.devx/technical-writing-guidelines/lists/#unordered-list)

P.S. The same goes for the remaining three list entries.

Author: I thought about this as a definition list. Changed it according to the guideline, thanks!

Reviewer: A definition list would also do. But I wasn't sure if definition lists are supported in your system.

- *"insertRowAbove"* / *"insertRowBelow"* / *"deleteRow"* - Manage rows
- *"insertColumnLeft"* / *"insertColumnRight"* / *"deleteColumn"* - Manage columns
- *"tableProperties"* / *"cellProperties"* - Open a pop-up window with table or cell customization options

## Manage Tables with the Context Menu

To enable table management with the context menu, set the [tableContextMenu](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableContextMenu/).[enabled](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableContextMenu/#enabled) property to **true**. In this demo, you can click the Enable Table Context Menu check box under the HtmlEditor to enable or disable the property. When it is enabled, right-click a table cell to open the context menu.

> [note] The context menu cannot be used to create new tables because it is only active within table boundaries. If you want users to create tables, make sure to add the *"insertTable"* item to the toolbar.

Reviewer: ```suggestion
[note] The context menu cannot be used to create new tables because it is active only within table boundaries. If you want users to create tables, make sure to add the *"insertTable"* item to the toolbar.
```

P.S. What do you mean by "active"? Available (that is, one can invoke the mentioned context menu only within the table boundaries)?

Author: I saw "only" moved back and forth may times, so I'm gonna keep it where it is and see what the editor got to say. Thanks for the other suggestions though!

Reviewer: You are welcome.

If you need to customize menu commands, override the **tableContextMenu**.[items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableContextMenu/items/) array. Refer to its description for a code example.

## Manage Tables Programmatically

> You can initiate table-related actions from code. Use the [getModules](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Methods/#getModulemoduleName) method to access the Table module. This module contains methods whose names match the toolbar item names listed above. Refer to the following topic for the complete method list and code examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md" target="_blank">DevExtreme Quill modules: Table</a>.

Reviewer: > initiate

Did you want to say "perform"? You just execute them programmatically, not just "prompt" or "start" (initiate).

> **from** code

**in** code. I guess now I finally understand where it came from. This is normally a very Russian thing to say "to do something **from** code". It's likely not a big mistake to say "from code" in English, however, "**in** code" sounds way more natural.

## Resize Tables

If users should be able to resize table rows and columns, set the [tableResizing](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/).[enabled](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#enabled) property to **true**. In this demo, you can use the Allow Table Resizing check box to change the property value.

> You can also set the **tableResizing**.[minColumnWidth](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#minColumnWidth) and **tableResizing**.[minRowHeight](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#minRowHeight) properties to limit minimum column width and row height. If you do not specify these properties, the width and height are limited by the cell content. You can see this behavior in this demo when you reduce column width or row height to small values.

Reviewer: ```suggestion
You can also set the **tableResizing**.[minColumnWidth](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#minColumnWidth) and **tableResizing**.[minRowHeight](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#minRowHeight) properties to limit the minimum column width and row height. If you do not specify these properties, the width and height are limited by the cell content. You can test out this behavior in this demo if you reduce the width or height of a column or row to the threshold values.
```

> DevExtreme HTML/Markdown Editor allows users to work with tables. This demo illustrates table resize support and table management with the toolbar, context menu, and in code.

Reviewer 2: - What does `to work with` imply?
- `table resize support` is a noun string. Is there such a thing as 'table resize support'?

Author: > What does to work with imply?

Create and manage tables.

> table resize support is a noun string. Is there such a thing as 'table resize support'?

I took it from our blog post approved by Ray:
https://community.devexpress.com/blogs/javascript/archive/2021/10/08/devextreme-html-markdown-editor-enhanced-table-support-and-soft-line-breaks-v21-2.aspx

Reviewer 2: 1st part - DevExtreme HTML/Markdown Editor allows users to create and manage tables.

2nd part - no comment

## Manage Tables with the Toolbar

To allow users to manage tables with the [toolbar](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/), add the following [items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/) to it:

- *"insertTable"* / *"deleteTable"*         
Manage a table.

- *"insertRowAbove"* / *"insertRowBelow"* / *"deleteRow"*       
Manage rows.

- *"insertColumnLeft"* / *"insertColumnRight"* / *"deleteColumn"*           
Manage columns.

- *"tableProperties"* / *"cellProperties"*          
Open a pop-up window with table or cell customization options.

## Manage Tables with the Context Menu

To enable table management with the context menu, set the [tableContextMenu](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableContextMenu/).[enabled](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableContextMenu/#enabled) property to **true**. In this demo, you can click the Enable Table Context Menu check box under the HtmlEditor to enable or disable the property. When it is enabled, right-click a table cell to open the context menu.

> [note] The context menu cannot be used to create new tables because it is only available within table boundaries. If you want users to create tables, make sure to add the *"insertTable"* item to the toolbar.

Reviewer 2: ```suggestion
[note] The context menu cannot be used to create new tables because it is only available within table boundaries. If you want users to create tables, add the *"insertTable"* item to the toolbar.
```

If you need to customize menu commands, override the **tableContextMenu**.[items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableContextMenu/items/) array. Refer to its description for a code example.

## Manage Tables Programmatically

You can perform table-related actions in code. Use the [getModules](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Methods/#getModulemoduleName) method to access the Table module. This module contains methods whose names match the toolbar item names listed above. Refer to the following topic for the complete method list and code examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md" target="_blank">DevExtreme Quill modules: Table</a>.

## Resize Tables

> If users should be able to resize table rows and columns, set the [tableResizing](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/).[enabled](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#enabled) property to **true**. In this demo, you can use the Allow Table Resizing check box to change the property value.

Reviewer 2: ```suggestion
If you want to allow users to resize table rows and columns, set the [tableResizing](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/).[enabled](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#enabled) property to **true**. In this demo, you can use the Allow Table Resizing check box to change the property value.
```

> You can also set the **tableResizing**.[minColumnWidth](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#minColumnWidth) and **tableResizing**.[minRowHeight](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#minRowHeight) properties to limit the minimum column width and row height. If you do not specify these properties, the width and height are limited by the cell content. You can test the latter behavior in this demo if you reduce column width or row height to small values.

Reviewer 2: ```suggestion
You can also set the **tableResizing**.[minColumnWidth](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#minColumnWidth) and **tableResizing**.[minRowHeight](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/#minRowHeight) properties to specify the minimum column width and row height. If you do not specify these properties, the width and height are determined by the cell content. This demo illustrates the latter behavior when you reduce column width or row height.
```

Author: > This demo illustrates the latter behavior when you reduce column width or row height.

It's important that you reduce the width or height greatly. If you reduce them just a bit, you won't see the behavior.

Reviewer 2: This demo illustrates the latter behavior when you reduce column width or row height to small values.
